doc/docs/learn/blocks.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+sidebar_label: Block Transitions
+---
+
+# Blocks
+
+Monomer applications must support both the usual Cosmos SDK application functionality and rollup-specific workflows.
+
+In order to acheieve this, Monomer defines a custom `Block` and block `Header` types, with helpers to generate representations that are consistent with both the Cosmos SDK and the EVM.
+
+```go
+type Block struct {
+	Header *Header
+	Txs    bfttypes.Txs
+}
+
+func (b *Block) ToEth() (*ethtypes.Block, error) {...}
+func (b *Block) ToCometLikeBlock() *bfttypes.Block {...}
+
+type Header struct {
+	ChainID    ChainID
+	Height     uint64
+	Time       uint64
+	ParentHash common.Hash
+	StateRoot  common.Hash
+	GasLimit   uint64
+	Hash       common.Hash
+}
+
+func (h *Header) ToComet() *bfttypes.Header {...}
+func (h *Header) ToEth() *ethtypes.Header {...}
+
+```
```

Comment:
Wdyt about removing this? Since our implementation is still changing quite a bit, I'd like to avoid risking the docs getting out of date.

Reply:
Probably better to remove, for those reasons. Will do.

I do think that callouts like graphics or code snippets help to cement the central points. Maybe a mini diagram such as:

![image](https://github.com/user-attachments/assets/5b59a349-3b64-47aa-90ae-14daa7080555)